packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -399,7 +409,26 @@ class _HelperErrorState extends State<_HelperError> with SingleTickerProviderSta
   }
 
   Widget _buildError() {
-    assert(widget.error != null || widget.errorText != null);
+    assert(
+      widget.error != null || widget.errorText != null,
```

Comment:
Might as well use the getter!

```suggestion
      _hasError,
```